components/raftstore/src/store/config.rs

```diff
@@ -250,16 +258,18 @@ impl Default for Config {
             local_read_batch_size: 1024,
             apply_batch_system: BatchSystemConfig::default(),
             store_batch_system: BatchSystemConfig::default(),
+            store_io_pool_size: 2,
```

Comment:
With multiple io threads, is it possible that raft logs from the same region are appending concurrently? And cause log appending out of order like some logs (with larger index) are appended before the log (with smaller index) in front of them.

Reply:
It's not possible. The io tasks must be handled in sequence.